Lib/argparse.py

```diff
@@ -1959,6 +1959,9 @@ def consume_positionals(start_index):
             for action, arg_count in zip(positionals, arg_counts):
                 args = arg_strings[start_index: start_index + arg_count]
                 start_index += arg_count
+                if action.nargs not in [PARSER, REMAINDER]:
```

Comment:
Two `if` conditions can be combined in one line using `and` operator: `if '--' in args and action.nargs not in [PARSER, REMAINDER]:`

Reply:
Hansraj, thanks for your comment.  But i don't think the two `if` should combined, because each judgement on behalf of each aspect.

Reply:
Ok, makes sense. We can resolve this comment. Sorry for the delay!

Reply:
NP, waiting maintainer's comment ;)